templates/documentation_layout.html

```diff
@@ -5,11 +5,13 @@
     <meta name="viewport" content="width=device-width" />
     <title>{{ page_title }}</title>
     <link rel="stylesheet" href="{{ unnest }}/index.css" type="text/css" />
+    <script src="{{ unnest }}/docs_config.js"></script>
```

Comment:
Does this file get automatically added by hexdocs? If so can we add a comment here to explain that.